tests/core/test_run.py

```diff
@@ -40,3 +42,38 @@ def test_create_single_input_channels_by_class_wo_credentials():
 
     assert len(channels) == 1
     assert channels[0].name() == "rest"
+
+
+async def test_load_agent_on_start_with_good_model_file(
+    trained_rasa_model, rasa_server, loop,
```

Comment:
You'll probably have to run the black formatting on this then
```suggestion
    trained_rasa_model: Text, rasa_server: Sanic, loop: AbstractEventLoop,
```